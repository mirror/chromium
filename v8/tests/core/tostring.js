function Point(x,y) {
  this.x = x;
  this.y = y;
}


Point.prototype.toString = function() {
  gc();
  return '(' + this.x + ',' + this.y + ')';  
};


print(new Point(16, 17));
print(new Point(5, 12));
