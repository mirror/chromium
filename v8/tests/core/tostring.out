(16,17)
(5,12)
